cfgov/unprocessed/css/organisms/mega-menu.less

```diff
@@ -409,6 +415,14 @@ body {
                     border-bottom: none;
                 }
             }
+
+            h3 {
+                margin-top: @grid_gutter-width / 2;
```

Comment:
There are some issues with the styling on smaller screens:

![screen shot 2017-07-26 at 15 14 28](https://user-images.githubusercontent.com/1044670/28639211-6a0c1982-7215-11e7-8171-91732200fe2b.png)

In the above screenshot, note how close the 2nd group title (Guides) is to the group above it. It should be more associated visually with its group. I think either this margin should increase, or a border should be added to the top of the first item in the following group, to segment the margins, or both.

You can also see a little extra space where a group continues into a second `ul`. This is due to [this rule in CF](https://github.com/cfpb/capital-framework/blob/canary/src/cf-core/src/cf-base.less#L379-L384), which should probably get overridden in the menu styling.